### PR TITLE
Cv-Styled

### PR DESCRIPTION
**Pull request by scrafy**

### CODE REVIEW

File: staff/joseluis-juste/cv-styled/Index.html [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Currículum Vitae</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <header class="header">
+        <div class="header-personal-img">
```
Comment (bcasasus) [inline staff/joseluis-juste/cv-styled/Index.html]: Hey! `<div>` it is not a semantic tag! Check the links below to know which tag fits better in your case.

- [HTML elements reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element)

- [HTML5 Semantic Elements](https://www.w3schools.com/Html/html5_semantic_elements.asp)

File: staff/joseluis-juste/cv-styled/Index.html [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+            <img src="./assets/logo.jpg" alt="">
+        </div>
+        <div class="header-title">
```
Comment (bcasasus) [inline staff/joseluis-juste/cv-styled/Index.html:17]: Careful here, you are using a `<p>` tag as a title. Don't you think that would be better to user a heading tag instead?

- [Heading Elements](https://developer.mozilla.org/ca/docs/Web/HTML/Element/Heading_Elements)

- [HTML Headings](https://www.w3schools.com/Html/html_headings.asp)

Comment (scrafy) [reply staff/joseluis-juste/cv-styled/Index.html:17]: Ok, fixed

File: staff/joseluis-juste/cv-styled/Index.html (hunk 1 above)
Comment (scrafy) [reply staff/joseluis-juste/cv-styled/Index.html]: Ok,fixed

**Pull request merged**